rlberry/manager/utils.py

```diff
@@ -14,3 +16,58 @@ def create_database(db_file):
         connection.close()
         return True
     return False
+
+
+def tensorboard_folder_to_dataframe(path_to_tensorboard_data):
+    """
+    Function to convert 'tensorboard log' to 'Panda DataFrames'.\
+    To convert the 'tensorboard log', the input must be must be the path to "the parent folder of all the training log" (path_to_tensorboard_data), and the 'events.out.tfevents' files have to be in this kind of path :
+      < path_to_tensorboard_data/algo_name/n_simu/events.out.tfevents.xxxxx >.
+
+    The output format is a dictionary.
+        key = tag (type of data).\
+        value = Panda DataFrame with the following structure (4 column) :
+            "name" = algo_name.\
+            "n_simu" = n_simu (seed).\
+            "x" = step number.\
+            "y" = value of the data.
```

Comment:
Look at the API doc, I dont think the newline "\" render well with the rst syntax used for docstring.

Reply:
done !